Core/MSBuild/MSBuild.cs

```diff
@@ -46,22 +46,21 @@ public void CallMSBuild (string msBuildPath, string arguments)
 
     var msBuildStartInfo = new ProcessStartInfo(msBuildPath, arguments);
     msBuildStartInfo.UseShellExecute = false;
-    msBuildStartInfo.RedirectStandardError = true;
     msBuildStartInfo.RedirectStandardOutput = true;
     msBuildStartInfo.WorkingDirectory = Environment.CurrentDirectory;
 
     var msBuildProcess = Process.Start(msBuildStartInfo);
+
+    var msBuildOutput = msBuildProcess!.StandardOutput.ReadToEnd();
+    
     msBuildProcess!.WaitForExit(c_msBuildProcessTimeout);
```

Comment:
Given the updated code location, we can drop the `damit` here in line 56 since we now trigger the null-error in line 54.